### PR TITLE
feat: added content length

### PR DESCRIPTION
**Pull request by PritishBudhiraja**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Description

Added content length in logs

## How did you test it?

Via checking the request headers content-length

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Comment by PritishBudhiraja** [comment]
https://github.com/juspay/hyperswitch-cloud/issues/5049

File: src/Utilities/PaymentHelpers.res [hunk 1]
```diff
@@ -948,6 +948,7 @@ let usePaymentIntent = (optLogger, paymentType) => {
       let uri = `${endpoint}/payments/${paymentIntentID}/confirm`
 
       let callIntent = body => {
+        let contentLength = body->String.length
```
Comment (vsrivatsa-juspay) [inline src/Utilities/PaymentHelpers.res:951]:
```suggestion
        let contentLength = body->String.length->Int.toString
```

File: src/Utilities/PaymentHelpers.res [hunk 2]
```diff
@@ -972,17 +973,17 @@ let usePaymentIntent = (optLogger, paymentType) => {
           handleLogging(
             ~optLogger,
             ~internalMetadata=loggerPayload,
-            ~value="",
+            ~value=contentLength->Int.toString,
```
Comment (vsrivatsa-juspay) [inline src/Utilities/PaymentHelpers.res:976]:
```suggestion
            ~value=contentLength,
```

File: src/Utilities/PaymentHelpers.res [hunk 3]
```diff
@@ -972,17 +973,17 @@ let usePaymentIntent = (optLogger, paymentType) => {
...
             (),
           )
         | _ =>
-          let _ = bodyArr->Array.map(((str, json)) => {
+          bodyArr->Array.forEach(((str, json)) => {
             if str === "payment_method_type" {
               handleLogging(
                 ~optLogger,
-                ~value="",
+                ~value=contentLength->Int.toString,
```
Comment (vsrivatsa-juspay) [inline src/Utilities/PaymentHelpers.res:986]:
```suggestion
                ~value=contentLength,
```

**Review by vsrivatsa-juspay — APPROVED** [review]

**Review by prafulkoppalkar — APPROVED** [review]

**Pull request merged**